test pr (added libSceNet dummy functions)

Addy2004 opened the pull request:
functions added:

sceNetSocket
sceNetSetsockopt
sceNetBind
sceNetListen
sceNetAccept

Addy2004 committed:
test commit

dummy functions added:
sceNetSocket
sceNetSetsockopt
sceNetBind
sceNetListen
sceNetAccept

Addy2004 committed:
Update main.yml

Addy2004 committed:
fixed workflow

red-prig commented:
@Addy2004 Declare correctly passed parameters, I won't accept placeholder functions without parameters

and468860 commented:
nop nid:libSceNet:3975D7E26524DEE9:sceNetConnect

Addy2004 committed:
Update ps4_libscenet.pas

more libSceNet dummy functions added:
sceNetRecv
sceNetSend
sceNetShutdown
sceNetSocketClose
sceNetGetsockname
sceNetNtohl
sceNetNtohs
sceNetEpollControl
sceNetResolverStartAton
sceNetResolverDestroy

correctly declared parameters also added

Addy2004 commented:
functions corrected(parameters):
sceNetSocket
sceNetSetsockopt
sceNetBind
sceNetListen
sceNetAccept

functions added:
sceNetRecv
sceNetSend
sceNetShutdown
sceNetSocketClose
sceNetGetsockname
sceNetNtohl
sceNetNtohs
sceNetEpollControl
sceNetResolverStartAton
sceNetResolverDestroy
